README.md

```diff
@@ -1,126 +1,198 @@
+Any modifications to your upstream chart like **adding the partner label** will be done in the `charts` directory.
 
-Modifying **Rancher modified** takes extra steps, as it requires modifications to be saved into patch files so that later it can retrieve the chart based on upstream chart and patch files.
+Add the partner label and required annotations in `Chart.yaml`:
 
-The step includes:
+```yaml
+annotations:
+  catalog.cattle.io/certified: partner
+  catalog.cattle.io/namespace: {CHART_NAMESPACE}       # Your chart's name is recommended
+  catalog.cattle.io/release-name: {CHART_RELEASE_NAME} # Your chart's name is recommended
+```
 
-1. Run `make CHART={CHART_NAME} prepare`
-   
-   This prepares `charts` with the current upstream chart and current patch. 
-   
-2. Change the version in `package.yaml`. If upstream chart needs to be updated, update url to point the latest chart. `packageVersion` also needs to updated.
+Save all your changes in the patch once you are done
 
-3. Make modification to your charts. 
+```
+make patch CHART={CHART_NAME}
+```
 
-4. Run `make CHART={CHART_NAME} patch`
- 
-   This will compare your current chart with upstream chart and generate the correct patch. 
-   
-5. Run `make CHART={CHART_NAME} clean`
-   
-   This will clean up the `charts` directory so that it won't committed.
+### 5. Add Overlay Files
 
-This repo provides a [workflow](./.github/workflows) that automatically uploads patch files and tarball of charts. Commit will only need to update `package/${chart-name}/charts` and make sure patches are 
-up-to-date with the latest chart. It also automatically build github pages to serve `index.yaml` and artifacts of charts.
+Any files you want to add on top of your upstream chart such as Rancher add-ons and logo (if you prefer keeping a local 
+copy), should be placed in the `overlay` directory.
 
-### Override existing Chart
+If this is a new chart, you will need the following:
 
-By defauly CI script doesn't allow changes to be made against existing chart. In order to make changes you have to bump chart version. There is a backdoor method to make changes to your existing chart without having to bump version. You can delete the tar.gz file you want to override and commit the change. Here is an example of [commit](https://github.com/rancher/dev-charts/commit/8be888076487e23a24121a532d25b9bf9ea936f3).
+- `app-readme.md` - Write a brief description of the app and how to use it. It's recommended to keep it short as the longer
+ `README.md` in your chart will be displayed in the UI as “detailed description”.
 
-### Helm repo index
+- `questions.yaml` - These are the questions displayed in the chart's installation page, which answers are used to configure 
+the chart by populating its `values.yaml`. Generally, the questions are focused on covering common use cases and include 
+default values so users can install the chart with little effort.
```

Comment:
```suggestion
- `questions.yaml` - Allows you to define a set of questions that user can provide answers to. These questions will be displayed on the chart's installation page to make it easier for a user to configure common use cases / set default values exposed by the chart's `values.yaml` so that users can install the chart with little effort.
```